_posts/memory-creation-overflow-bug.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title:  "Solidity Memory Array Creation Overflow Bug"
+date:   2020-04-02
+author: Solidity Team
+---
+
+# Solidity Memory Array Creation Overflow Bug Announcement
+
+On the 28th of March a bug in the Solidity code generator was reported through the
+<a href="https://bounty.ethereum.org/" target="_blank">Ethereum Foundation Bounty program</a>,
```

Comment:
```suggestion
<a href="https://bounty.ethereum.org/">Ethereum Foundation Bounty program</a>,
```
Why force?

Reply:
I just copied that from 2017-05-03-solidity-optimizer-bug.md, not sure why it was there, I'll remove.